src/rules/font-family-name-quotes/index.js

```diff
@@ -54,15 +52,22 @@ export default function (expectation) {
     if (!validOptions) { return }
 
     root.walkDecls(/^font-family$/i, decl => {
```

Comment:
Since this uses your nifty new `findFontFamily`, can we also make it check `font` declarations, or were you saving that for later?